src/main/java/backend/view/sponsoring/SponsoringView.kt

```diff
@@ -37,19 +35,19 @@ class SponsoringView() {
 
     var billableAmount: Double? = null
 
-    var teamDistance: Double? = null
+    var teamsTotalDistance: Double? = null
```

Comment:
I'm unsure whether the total team distance is enough in here. If we need it on a per team base, we might need to extend the `teams` map to not only include the name as the value but also each team's individual distance.